src/proxy.conf.json

```diff
@@ -1,6 +1,6 @@
 {
   "/api": {
-    "target": "http://localhost:8090",
+    "target": "http://localhost:18120",
```

Comment:
I don't think it's correct, it should remain the "8090" as it was

Reply:
yop, just don't include it in the commits (I do it like that)